spec/presenters/payroll_data_csv_row_spec.rb

```diff
@@ -0,0 +1,61 @@
+require "rails_helper"
+
+RSpec.describe PayrollDataCsvRow do
+  subject { described_class.new(claim) }
+  let(:claim) { build(:claim) }
+
+  describe "to_s" do
+    let(:row) { CSV.parse(subject.to_s).first }
+    let(:start_of_month) { Date.today.at_beginning_of_month }
+
+    let(:claim) do
+      build(:claim, :submittable,
+        payroll_gender: :female,
+        date_of_birth: Date.new(1980, 12, 1),
+        student_loan_plan: StudentLoans::PLAN_2,
+        bank_sort_code: "001122",
+        bank_account_number: "01234567",
```

Comment:
I _think_ if you wrap a number in quotes, Excel will treat it as a string. Or maybe it's `="001122"`. This was the solution I originally implemented to (incompletely) deal with formulae in values.

Reply:
I think the idea is for this CSV to be directly consumed without user editing so the value in Excel shouldn't matter. I think CSV parsers should preserve the leading zeros, I know the Ruby one does: 

```
irb(main):004:0> CSV.parse("001,XYZ,01234567")
=> [["001", "XYZ", "01234567"]]
```

Reply:
Yeah agreed. I guess we'll need to rely on guidance to tell people not to open the CSV and resave it.

Reply:
Yeah I think this is right. If you just "double click" a CSV file and let Excel open it, there is little we can do to prevent the leading zero's being dropped. The way to open the CSV and preserve the leading zeros is to use the File > Import feature and tell Excel how to handle specific columns.